File: VECTOAux/VectoAuxiliaries/Hvac/IBus.vb

Namespace Hvac

    Public Interface IBus

        ReadOnly Property Id As Integer

        Property Model As String
        Property FloorType As String
        Property EngineType As String
        Property LengthInMetres As Double
        Property WidthInMetres As Double
        Property HeightInMetres As Double
        Property RegisteredPassengers As Integer
        Property IsDoubleDecker As Boolean

    End Interface

End Namespace

File: VectoCore/VectoCore/Models/BusAuxiliaries/DownstreamModules/Impl/HVAC/IBus.cs
namespace TUGraz.VectoCore.Models.BusAuxiliaries.DownstreamModules.Impl.HVAC
{
	public interface IBus
	{
		int Id { get; }

		string Model { get; set; }
		string FloorType { get; set; }
		string EngineType { get; set; }
		double LengthInMetres { get; set; }
		double WidthInMetres { get; set; }
		double HeightInMetres { get; set; }
		int RegisteredPassengers { get; set; }
		bool IsDoubleDecker { get; set; }
	}
}
